docs/sources/enterprise/auditing.md

```diff
@@ -148,11 +149,15 @@ max_file_size_mb = 256
 
 ### Loki exporter
 
-Audit logs are sent to a [Loki](/oss/loki/) service.
+Audit logs are sent to a [Loki](/oss/loki/) service, through HTTP or gRPC.
+
+> The HTTP option for the Loki exporter is only available in Grafana Enterprise v7.4+.
 
 ```ini
 [auditing.logs.loki]
-# Set the URL for writing logs to Loki
+# Set the communication protocol to use with Loki (can be grpc or http)
+type = grpc
+# Set the URL for writing logs to Loki (format must be host:port)
```

Comment:
```suggestion
# Set the address for writing logs to Loki (format must be host:port)
```

Minor since people call non-URLs a URL all the time, but if we document this as an address rather than as a URL, someone who knows the formal definition of a URL is potentially slightly less confused.

Reply:
Agree!! 💯